### ISSUE TITLE
SVG component with a linear gradient does not render as expected in the Tabs component

### ISSUE THREAD

**Issue by inglkruiz** [ant-design/ant-design#42528]
### Reproduction link

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/basic-usage-deprecated-syntactic-sugar-antd-4-24-10-forked-8nxtq4?file=/demo.tsx)

### Steps to reproduce

1. Run the codesandbox provided
2. Change tabs between 1 and 2

### What is expected?

SolanaLogo (SVG component) in the Tab 2 should render as is render in the Tab 1

### What is actually happening?

SolanaLogo (SVG component) in the Tab 2 loses the linear gradient. 

| Environment | Info |
| --- | --- |
| antd | 4.24.10 |
| React | react |
| System | N/A |
| Browser | Any Chromium based  |

---
It also happens in v5 > https://codesandbox.io/s/basic-antd-5-5-0-forked-scv2vs?file=/demo.tsx

**Comment by thomaswinckell** [comment]
Your SVG contains ids. When you are using them twice in this particular case, there's a conflict.

I've been able to reproduce your example without using Tabs here : https://codesandbox.io/s/basic-usage-deprecated-syntactic-sugar-antd-4-24-10-forked-ox4r54?file=/demo.tsx

If you "display: none" the first SVG, then it looks like the SVG definitions are "erased" by the browser and the second SVG that is using those definitions can't process them.

Your issue can be solved by making the defs ids uniques. I did it using your example : https://codesandbox.io/s/basic-usage-deprecated-syntactic-sugar-antd-4-24-10-forked-s7blyd?file=/demo.tsx

You might want to fill an issue in the [rc-tabs](https://github.com/react-component/tabs) project as ant is using this package.
